.travis.yml

```diff
@@ -12,7 +12,17 @@ env:
   - TOXENV=py33-1.8
   - TOXENV=py34-1.8
   - TOXENV=py34-1.9
+  - TOXENV=py34-1.10
+  - TOXENV=py34-1.11
   - TOXENV=py34-master
+  - TOXENV=py35-1.9
+  - TOXENV=py35-1.10
+  - TOXENV=py35-1.11
```

Comment:
https://docs.travis-ci.com/user/languages/python/#Choosing-Python-versions-to-test-against

"Travis CI supports Python versions 2.6, 2.7, 3.2, 3.3, 3.4, 3.5, 3.6 as well as recent development versions."